k8s/agentcube-apiserver.yaml

```diff
@@ -89,6 +95,11 @@ spec:
             - name: http
               containerPort: 8080
               protocol: TCP
+          env:
+            - name: REDIS_ADDR
+              value: "127.0.0.1:6379"
```

Comment:
Nit: i guess this is a placeholder, please update using a var when we support helm